src/prism.rs

```diff
@@ -272,9 +252,7 @@ impl<T: Data> Prism<Option<T>, T> for OptionSome {
     }
 }
 
-prism!(OptionNone: Option<String> => None);
```

Comment:
Thanks for reverting this weird thing as well, it tripped me up before.

Reply:
Yeah. It was work in progress, but I never hat the time to finish it.